solr-ingest/src/main/java/edu/unc/lib/dl/data/ingest/solr/filter/SetAccessStatusFilter.java

```diff
@@ -105,7 +106,7 @@ public void setObjectAclFactory(ObjectAclFactory oaf) {
             status.add(FacetConstants.PUBLIC_ACCESS);
         }
 
-        if (objPrincRoles.size() > 0) {
+        if (objPatronRoles.size() > 0) {
```

Comment:
After checking in with Julie about this, the tag should only appear if the patron permissions are set to something other than the default. 

For folders/works/files, the default would be either `objPatronRoles.size() == 0` OR the `authenticated` and `everyone` groups both set to `canViewOriginals`.

For collections, it has to work a little different, we would show `patron-settings` if `objPatronRoles.size() < 2`, or either`authenticated` or `everyone` groups had any value other than `canViewOriginals`.

And we can skip this tag altogether for Units.